modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/GetLinterCustomRulesThroughThePublisherRestAPITestCase.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ *   Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an
+ *   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *   KIND, either express or implied.  See the License for the
+ *   specific language governing permissions and limitations
+ *   under the License.
+ *
+ */
+
+package org.wso2.am.integration.tests.publisher;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.testng.annotations.*;
```

Comment:
Shall we remove the * import ?

Reply:
Change added in [dce5303](https://github.com/wso2/product-apim/pull/12971/commits/dce5303624fbab0df523d5d4a63a9f2152868a8b)